src/app.scss

```diff
@@ -0,0 +1,6 @@
+.main {
+  background-color: $primary-color;
+  padding-left: rem(35);
```

Comment:
También se puede poner en una única propiedad: `padding: rem(49) 0 rem(35) rem(35);`
https://www.w3schools.com/css/css_padding.asp
Up to youuuu :)